xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml

```diff
@@ -967,20 +970,35 @@ require(['jquery'], function($) {
 }
 
 .admin-menu a.panel-heading {
-  display: block;
+  display: flex;
   font-size: 100%;
 }
+
+.admin-menu a.panel-heading span {
+  flex-grow: 1;
+}
+
 .admin-menu a.panel-heading:not(.collapsed) {
   background-color: $theme.menuBackgroundColor;
   color: $theme.menuLinkColor;
 }
+
+/* Rotate the carret on collapse. */
+.admin-menu a.panel-heading.collapsed &gt; *:last-child {
+  transform: rotate(90deg);
```

Comment:
Interesting, I thought you were using two different icons, but it's same with rotation. Do you know if that's what we're using in other places?

Reply:
We don't have a caret-left icon in the IconTheme, so it's pretty common to do so.

There's some places where we use two icons, but that's with `caret-right` and `caret-down` together.

I checked a bit further than before and found:
* This `more option` dropdown at the bottom of the admin export tab, IMO we should update this :)
![image](https://github.com/xwiki/xwiki-platform/assets/28761965/0a6457f5-ca79-49d8-a52e-478de5a21cf9)
* This `Local history` dropdown in extension history
![image](https://github.com/xwiki/xwiki-platform/assets/28761965/f1f927c6-b034-457d-b318-1810230d14bd)
* In the `displayJobStatus` velocity macro, on a dropdown for the logs:
![image](https://github.com/xwiki/xwiki-platform/assets/28761965/fffbb847-317f-4848-b1fe-1cbb680bab72)

... and in a few other places.
Most of those places have their own classes.
I'll spend some time comparing them and proposing a design pattern to make it easier to maintain on the long run :)